exporter/collector/integrationtest/testcases/conversion.go

```diff
@@ -152,13 +152,13 @@ func convertHistogram(h pmetric.Histogram) metricdata.Aggregation {
 	if h.DataPoints().Len() == 0 {
 		return nil
 	}
-	agg := metricdata.Histogram{
+	agg := metricdata.Histogram[float64]{
```

Comment:
As a note, I think it is fine that we are only handling float64 histograms here, because 
1. this code is just handling testcases.
2. `Sum` is the only value used here that can be int or float in `metricdata.Histogram[N]` (we don't read [Min or Max](https://github.com/open-telemetry/opentelemetry-go/blob/e0852d609c4a4205d550e2de45afdbf80d43557b/sdk/metric/metricdata/data.go#L130-L134)) and `Sum()` always returns a float64, so there shouldn't be any runtime issues with this.
3. We [force `float64`](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/620/files#diff-0a6b748481b08d3c4a7c5db8799a3b134f70e26fa97566ec8e0c4543d81783e7L615-R630) when we handle Sum anyway, which is consistent with the behavior prior to this update and therefore not breaking.

However, we should follow this up with support for `int64` histograms. Otherwise, users who switch to the GCP exporter might see slightly different values since we force floats in the output calculation.  That would include adding a test case for an int64 histogram, which we don't appear to have currently. Will make a separate issue to track this.